codebasin/config.py

```diff
@@ -105,7 +105,7 @@ def __call__(
             template = string.Template(self.format)
             split_values = [template.substitute(value=v) for v in split_values]
         if self.dest == "passes":
-            passes = getattr(namespace, self.dest)
+            passes = getattr(namespace, "_passes")
```

Comment:
Just making sure that `getattr` will never return `None` if `_passes` is undefined in `namespace`. Will that ever happen?

Same comment applies to L147

Reply:
I wasn't sure what would happen here, so I wrote a small test:
```python
import argparse
ns = argparse.Namespace()
getattr(ns, "_passes")
```

...which crashes with:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Namespace' object has no attribute '_passes'
```

So we won't get a silent failure where `None` propagates and weird things happen; the `AttributeError` should be caught by the top-level `try`/`except` (in `__main__.py`) and get printed as an error.

Does this answer your question?

Reply:
Sounds good to me!